.github/workflows/checkin.yml

```diff
@@ -0,0 +1,31 @@
+name: pull-request
+on:
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+      - uses: actions/checkout@v2
+      - name: Set up node 
+        uses: actions/setup-node@v1
```

Comment:
why does shadow need nodejs ?  this is a bit weird.

Reply:
Heh, it doesn't.  I'm copy/pasting on my first attempt.

Sorry about wasting your time - probably best to wait until tomorrow to bother reviewing :)  I'm just trying to get it to trigger anything at all.